src/tests/ftest/util/agent_utils.py

```diff
@@ -65,8 +161,27 @@ def run_agent(basepath, server_list, client_list=None):
     sessions = {}
     user = getpass.getuser()
 
-    # if empty client list, 'self' is effectively client
-    client_list = include_local_host(client_list)
+    # if empty client list, 'test' is effectively client
+    if client_list == None:
+        client_list = include_local_host(client_list)
+    client_count = len(client_list)
+
+    # Create the DAOS Agent configuration yaml file to pass
+    # with daos_agent -o <FILE_NAME>
+    agent_yaml = os.path.join(test.tmp, "daos_agent.yml")
+    agent_config = DaosAgentConfig()
+    agent_config.get_params(test)
+    agent_config.hostlist.value = client_list
+
+    access_point_list = []
+    for index in range(client_count):
+        access_point = client_list[index] + ":" + str(agent_config.port)
+        access_point_list.append(access_point)
+    agent_config.access_points.value = access_point_list
```

Comment:
"client_list"? I've posted my previous mail reply on "access_points" to DAOS-3624. Could you take a look, please?

Reply:
Will use the 1st server from the server_list as access_point.
ie:    
Client boro-50:
        access_points: server_list[0]:10001